src/simulators/statevector/statevector_state.hpp

```diff
@@ -40,10 +40,10 @@ enum class Snapshots {
 // QubitVector State subclass
 //=========================================================================
 
-template <class data_t = complex_t*>
-class State : public Base::State<QV::QubitVector<data_t>> {
+template <class statevec_t>
```

Comment:
Can you change it to still has the original default class:

```c++
template <class statevec_t = QV::QubitVector<complex_t*>>
```

Reply:
Thanks. I added a default class for the template and remove class specification in callers.